### PR TITLE
Remove skips not used by any lane 

### PR DESCRIPTION
**Pull request by maya-r**
We previously had a lot of trouble with a lane running on "kubernetes in docker" (kind).
The only remaining use for it is the SR-IOV lane, that only runs a subset of tests.
Remove no longer relevant skips - the issues referenced by the skip are already closed.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by ashleyschuett — APPROVED** [review]

**Review by EdDev — COMMENTED** [review]
I am unsure that we need these helpers now.
The only `kind` cluster is SR-IOV and non of these tests are expected to run there.

This may have been a leftover from when removing the IPv6 kind lane.

**Comment by EdDev** [comment]
> EDIT: Can you perhaps just try to remove these?

Please unhold if it does not work.
/hold

**Review by EdDev — APPROVED** [review]
Thank you!

/unhold

**Comment by maya-r** [comment]
/hold
Sending a cautious email to kubevirt-dev and waiting 2 weeks before merging.

**Review by enp0s3 — CHANGES_REQUESTED** [review]
@maya-r Hi. Thank you for the PR. Just to be on the safe side- if someone would try to run the e2e suite on an external cluster (we allow this by setting `KUBEVIRT_PROVIDER=external`) can you please make sure we have a warning in the documentation about using KIND provider in combination with these tests?

**Comment by maya-r** [comment]
Apparently that file comes from kubevirtci.

**Review by enp0s3 — APPROVED** [review]
/approve

**Comment by maya-r** [comment]
Fixed merge conflict.
Will abuse this almost-no-changes pull request to test an optional lane before enabling it. Sorry for the noise.

/test pull-kubevirt-e2e-k8s-1.20-rook-ceph

**Comment by maya-r** [comment]
No replies, assuming it's fine.
/hold cancel

**Review by EdDev — APPROVED** [review]

**Pull request merged**
